src/pyprediktormapclient/opc_ua.py

```diff
@@ -141,7 +141,7 @@ def get_live_values_data(
             )
         df = self.expand_props_vars(obj_dataframe)
         name_column = [
-            x for x in df if x in ["DisplayName", "DescendantName", "AncestorName"]
+            x for x in df if x in ["Name", "DisplayName", "DescendantName", "AncestorName"]
```

Comment:
Should it be both "Name" and "DisplayName"?

Reply:
Will be removed when we don't need the compatibility. Making changes in the ModelIndex-part first and then the OPC UA later